docs/blog/design-philosophy/index.qmd

```diff
@@ -0,0 +1,245 @@
+
+Zooming ahead about a thousand years, you start to see more systematically structured tables. Here's a photo of a cuneiform tablet that was originally from Mesopotamia (from the Temple of Enlil at Nippur, ca. 1850 BCE)[^6].
+
+![](./nippur_cuneiform_tablet.png)
+
+<p style="font-size: 12px;">Cuneiform tablet, temple of Enlil at Nippur, (CBS 3323, University of Pennsylvania).</p>
+
+You can see right away that there is a more regular grid and, if you probe deeper, there are more similarities than differerences with the tables of today. This contains sources of revenue and monthly disbursements for nearly 50 temple personnel. While it can be difficult to see it, you should take it on good authority[^7] that the following table elements are present:
+
+- column headings (month names) and row titles (names/professions of individuals).
+- cells with no information (look at the blank or smooth cells along rows)
+- numerical values in the cells
+- subtotals for each individual are every six months
+- grand totals adjacent to row labels
+- annotations with explanatory notes under those columns containing totals
+
+Later on, tables were less inscribed on clay and more on wax tablets, papyrus, paper, and eventually tables became computerized. What a trip it has been for tables!
+
+### Midcentury modern tables
+
+There's a prehistory to modern, computerized tables and perhaps the best period for tables was around the middle of the 20th century. Technologies for table (and surrounding document) preparation included the offset printer, the typewriter, and varitype[^8] (my favorite). While constrained by the limited space available on pages, tabular design at this point had many solutions for either fitting tables into single, variously-sized pages or distributing the table content across multiple pages.
+
+While there were knowledgeable practictioners who could well perform the task of typesetting tables in books, there wasn't much in the way of books about tabular presentation and all the attendant theoretical and practical approaches one might take. To remedy this (but more likely to serve as an internal tool for the organization), the United States Bureau of the Census published their *Manual of Tabular Presentation*[^9]. It is truly a remarkable work and one can readily plumb the considerable depths of this monograph for its excellent recommendations on tabular design. It spares no detail on how the department classifies the different parts of a table, describing the function of each part in rigorous detail. Moreover, it is easy to read and thankfully provides many examples in visual form. Through its many hundreds of paragraphs, the authors make strong recommendations on what to do (and what *not* to do) for many tabulation scenarios.
```

Comment:
You primed the pipe for people to expect great things of this era. Just punch them in the face with the manual of tabular presentation! I tried to cut out redundant sentence, and connect some pieces directly together.

```suggestion
Another reason this was a great period for tables is the publication of the *Manual of Tabular Presentation*[^9] by the United States Bureau of the Census. It is truly a remarkable work, which spares no detail on how the department classifies the different parts of a table, describing the function of each part in rigorous detail. Moreover, it is easy to read, thanks to its many visual examples. Throughout its hundreds of pages, the authors make strong recommendations on what to do (and what *not* to do) for many tabulation scenarios.
```